lib/protocol/collection/item_collection.ts

```diff
@@ -117,11 +117,25 @@ export class ItemCollection extends MutableCollection<SNItem> {
       /** Filtered content type map */
       const filteredCTMap = this.filteredMap[contentType]!;
       const sortedElements = this.sortedMap[contentType]!;
+
+      const currentIndex = filteredCTMap[element.uuid];
+      const currentElement = !isNullOrUndefined(currentIndex) ? sortedElements[currentIndex] : undefined;
+      if (currentElement?.errorDecrypting) {
+        if (element.errorDecrypting) {
+         /** if both elements are encrypted skip compare */
+          sortedElements[currentIndex] = element;
+          continue;
```

Comment:
It looks like

```
          sortedElements[currentIndex] = element;
          continue;
```

is repeated in both if/else conditions, so can be factored out.